### DISCUSSION TITLE
Change start metod Services:  Spooler (for printing) from Manual to Automatic

### DISCUSSION THREAD

**Discussion by Taogunka** [memstechtips/UnattendedWinstall#181] [Q&A] [answered]
Hi. I know that unattended.xml file is easily customizable for users, but wouldn't it be better to set the "Spooler" process start to "Automatic" from "Manual" mode by default? 
(for Printing and Printer installing)

**Comment by rautamiekka** [comment, marked as answer]
You can just go put it back on, like I did recently on someone else's machine.
